packages/ecs-agent/0001-bottlerocket-default-filesystem-locations.patch

```diff
@@ -49,9 +49,9 @@ index 080f18c4d..ef14970ce 100644
  	// defaultAuditLogFile specifies the default audit log filename
 -	defaultCredentialsAuditLogFile = "/log/audit.log"
 +	defaultCredentialsAuditLogFile = "/var/log/ecs/audit.log"
- 	// DefaultTaskCgroupPrefix is default cgroup prefix for ECS tasks
- 	DefaultTaskCgroupPrefix = "/ecs"
  
+ 	// defaultRuntimeStatsLogFile stores the path where the golang runtime stats are periodically logged
+ 	defaultRuntimeStatsLogFile = `/log/agent-runtime-stats.log`
```

Comment:
This likely needs to change to `/var/log/ecs/agent-runtime-stats.log` and may potentially need to get added to the logdog config too.

Reply:
Thanks, good catch.